### ISSUE TITLE
A MFA 1.1.0 beta 2 ISSUE

### ISSUE THREAD

**Issue by cuongnguyengit** [MontrealCorpusTools/Montreal-Forced-Aligner#218]
Initializing training for sat1...
Initializing speaker-adapted triphone training...
Initialization complete!
  0%|                                                                                                   | 0/34 [00:00<?, ?it/s]Traceback (most recent call last):
  File "aligner/command_line/train_and_align.py", line 131, in <module>
  File "aligner/command_line/train_and_align.py", line 70, in align_corpus
  File "aligner/aligner/trainable.py", line 81, in train
  File "aligner/trainers/sat.py", line 122, in train
Exception: There was an error training in iteration 1, please check the logs.
[24756] Failed to execute script train_and_align

Los is here:

ERROR (gmm-est[5.4.247~1-2148]:Read():kaldi-vector.cc:1223) Failed to read vector from stream. EOF while trying to read vector. File position at start is -1, currently -1

Please tell me how to fix this error? Thanks

**Comment by ronggong** [comment]
Similar problem here:
### Subject of the issue
Initialization complete!
100%|| 34/34 [01:41<00:00,  2.98s/it]
Initializing training for lda...
Initializing training for lda...
Initialization complete!
Initialization complete!
100%|| 34/34 [07:38<00:00, 13.49s/it]
Initializing training for sat1...
Initializing speaker-adapted triphone training...
Initialization complete!
  0%|                                                                                                                                                                                              | 0/34 [00:00<?, ?it/s]Traceback (most recent call last):
  File "aligner/command_line/train_and_align.py", line 131, in <module>
  File "aligner/command_line/train_and_align.py", line 70, in align_corpus
  File "aligner/aligner/trainable.py", line 81, in train
  File "aligner/trainers/sat.py", line 122, in train
Exception: There was an error training in iteration 1, please check the logs.
[16400] Failed to execute script train_and_align

### Your environment
Description:    Ubuntu 18.04.5 LTS

The errors in /Documents/MFA/wav_lab/sat1/log/acc.1.0.log 
```
ERROR (gmm-acc-stats-ali[5.4.247~1-2148]:SequentialTableReader():util/kaldi-table-inl.h:860) Error constructing TableReader: rspecifier is scp:/Documents/MFA/wav_lab/corpus_data/subset_10000/features_mfcc_cmvn_lda_fmllr.0.scp
```

It looks for "features_mfcc_cmvn_lda_fmllr.0.scp" in subset_10000 folder, but it can't find it. The .scp with the same name found in split8 folder when doing fmllr

```
transform-feats --utt2spk=ark:/Documents/MFA/wav_lab/corpus_data/split8/utt2spk.0 ark:/Documents/MFA/wav_lab/sat1/trans.0 scp:/Documents/MFA/wav_lab/corpus_data/subset_10000/features_mfcc_cmvn_lda.0.scp ark,scp:/Documents/MFA/wav_lab/corpus_data/split8/features_mfcc_cmvn_lda_fmllr.0.ark,/Documents/MFA/wav_lab/corpus_data/split8/features_mfcc_cmvn_lda_fmllr.0.scp
```

**Comment by mmcauliffe** [comment]
These issues should be resolved in 2.0 alpha version (https://montreal-forced-aligner.readthedocs.io/en/latest/installation.html), so I'm going to close this issue.  2.0 revamped a lot of the training schemes that 1.1 originally introduced and fixed a whole host of bugs, so it should work much better in 2.0.
